### DISCUSSION TITLE
Customize Relation Node Template

### DISCUSSION THREAD

**Discussion by jeremyzilar** [formfcw/directus-extension-flexible-editor#30] [Q&A] [answered]
Is it possible to customize the template for linked records in the Flexible Editor?

As an editor of a publication who is editing a few hundred articles each month, it would be nice to see which things we are embedding in our articles while making overall edits to the page.

<img width="848" alt="image" src="https://github.com/formfcw/directus-extension-flexible-editor/assets/395641/2fbf77a1-d5bd-424c-96e9-63b1d1db380b">

**Comment by formfcw** [comment, marked as answer]
Of course. Set the display for your related collection.

**Reply by jeremyzilar** [reply]
Ahh, the Display seems to work for one of the multiple related collections (images, contributors).
